### ISSUE TITLE
Support for Cppcheck-2.6

### ISSUE THREAD

**Issue by cvaliere** [jbenden/vscode-c-cpp-flylint#148]
There has been a new release v2.6 on cppcheck

When using this version with your extension, files "cppcheck-addon-ctu-file-list" are created and don't disappear; and also some MISRA are not detected

(I can try to give more information if you need, but I think that if you install the v2.6 of cppcheck, you'll see what's happening)

**Comment by ben-edna** [comment]
I've resolved this (for now) by specifying a build-dir to at least keep the files out of my project. (make sure the directory exists).

```json
  "c-cpp-flylint.cppcheck.extraArgs": [
        "--cppcheck-build-dir=C:\\Tools\\cppcheck\\build-dir"
    ],
```

**Comment by danmar** [comment]
I am the maintainer of Cppcheck. I think this is more a Cppcheck problem. It shouldn't leave the ctu artifacts on disk.

I have the intention to work on it but I can't say when it will be fixed.
